### PR TITLE
Update the shopping cart regardless of AA sync success

### PR DESCRIPTION
**Pull request by daisieh**

### CODE REVIEW

File: dspace/modules/xmlui/src/main/java/org/dspace/app/xmlui/aspect/submission/workflow/actions/processingaction/ReAuthorizationCreditActionXMLUI.java [hunk 1]
```diff
@@ -106,16 +106,16 @@ private String submitCredit(){
...
-                            Date date= new Date();
-                            shoppingCart.setPaymentDate(date);
-                            shoppingCart.update();
                             success = AssociationAnywhere.tallyCredit(context, metadataValues[0].value, DOIIdentifierProvider.getDoiValue(item));
                             paymentSystemService.sendPaymentApprovedEmail(context, workflowItem, shoppingCart);
                         }catch (Exception e)
                         {
-                            paymentSystemService.sendPaymentErrorEmail(context, workflowItem, shoppingCart,"problem: credit not tallied successfully. \\n \\n " + e.getMessage());
+                            paymentSystemService.sendPaymentErrorEmail(context, workflowItem, shoppingCart,"problem: credit not tallied successfully. \n \n " + e.getMessage());
                             log.error(e.getMessage());
```
Comment (ryscher) [inline dspace/modules/xmlui/src/main/java/org/dspace/app/xmlui/aspect/submission/workflow/actions/processingaction/ReAuthorizationCreditActionXMLUI.java:119]: change to

log.error(e);

Comment (daisieh) [reply dspace/modules/xmlui/src/main/java/org/dspace/app/xmlui/aspect/submission/workflow/actions/processingaction/ReAuthorizationCreditActionXMLUI.java:119]: If I did that, we wouldn't get the emails at the help desk.

Comment (ryscher) [reply dspace/modules/xmlui/src/main/java/org/dspace/app/xmlui/aspect/submission/workflow/actions/processingaction/ReAuthorizationCreditActionXMLUI.java:119]: Leave the email line, just make the logging more verbose by passing the entire exception.

**Comment by ryscher** [comment]
Looks good, compiles and runs. 

It's difficult to test this directly, since it depends on difficult-to-recreate conditions, but it's minor enough that it's ok to push out without further testing.

**Pull request merged**